### ISSUE TITLE
EPG update every 5 minutes

### ISSUE THREAD

**Issue by lama18** [xbmc/xbmc#15430]
### Describe the bug
EPG update runs every 5 minutes , despite the setting (defaulted with 120 min). Getting guide for many channels is an high cost operation for the backend and can get more than 5 minutes to retrieve the range of data. Is the reason of the setting and its default value (120 min).

## Expected Behavior
When you realize that one EPG file can have, for example, 10 MB, it will be downloaded for 250+ days, which is 2.5 GB + completely unnecessary.

 In this case, a check would be sufficient, for example, once a day / at startup / if the data ends.

## Actual Behavior
An EPG update bumps the backend every 5 min.

## Possible Fix
Check only once a day / at startup / if the data ends / Maybe just respect the settings in PVR

### To Reproduce
1.  Open Kodi

### Screenshots 
Here are some links or screenshots to help explain the problem:
![image](https://user-images.githubusercontent.com/20426629/52272993-c621d180-2948-11e9-99ca-db02c11213f1.png)
![image](https://user-images.githubusercontent.com/20426629/52273005-d043d000-2948-11e9-8da0-5e804f667320.png)

*Log from SQL where is access from Kodi every 5 minutes for download XMLTV file.
*XMLTV data on 3-5 days

### Your Environment
Used Operating system:
 - [ x] Android
 - [ ] iOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Raspberry-Pi
 - [x ] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version: 18

**Comment by ksooo** [comment]
Please post feature requests in the forum. This is an issue tracker.

**Comment by DaVukovic** [comment]
This is expected behaviour and as said multiple times now, it can be adjusted by the advancedsettings.xml

>  at startup
makes no sense for users who don't turn off their devices and leave them running for 24/7

> once a day
what if the EPG changes during the day and Kodi doesn't fetch the new data. So you will end up with wrong EPG information and users will complain about that.

> if the data ends
same as for "once a day"

Especially for those options the advancedsettings.xml exists that users are able to adjust it in the way they like it

**Comment by lama18** [comment]
![image](https://user-images.githubusercontent.com/20426629/52273739-02eec800-294b-11e9-8371-758eb869555e.png)

What is this setting for? If the timer respected this, it would be great. Regardless of the changeset advancementtings.xml, which is not very user friendly

**Comment by DaVukovic** [comment]
https://kodi.wiki/view/Settings/TV/Guide#Update_interval

Going to close that now. As those discussions should be done at the forums. Please do not re-open a new issue for that.
